charts/newrelic-k8s-metrics-adapter/templates/apiservice/job-patch/psp.yaml

```diff
@@ -2,13 +2,13 @@
 apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice
+  name: {{ template "common.naming.fullname" . }}-apiservice
   annotations:
     "helm.sh/hook": pre-install,pre-upgrade,post-install,post-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
   labels:
-    app: {{ template "newrelic-k8s-metrics-adapter.name" . }}
-{{ include "newrelic-k8s-metrics-adapter.labels" . | indent 4 }}
+    app: {{ template "common.naming.name" . }}
+{{ include "common.labels" . | indent 4 }}
```

Comment:
I saw it in different places, but I would prefer to have it as:

```
  labels:
    app: {{ template "common.naming.name" . }}
    {{- include "common.labels" . | nindent 4 -}}
```

In this was it is indented correctly automatically

(NIT about style, can be ignored)

Reply:
I agree about you, I didn't fix it and it's a good nit.

Reply:
done